.buildkite/nightly-benchmarks/nightly-pipeline.yaml

```diff
@@ -83,12 +117,52 @@ steps:
           podSpec:
             <<: *common_pod_spec
             containers:
-              - image: ghcr.io/huggingface/text-generation-inference:2.1 
+              - image: nvcr.io/nvidia/tritonserver:24.07-trtllm-python-py3
                 <<: *common_container_settings
+                env:
+                  - name: VLLM_USAGE_SOURCE
+                    value: ci-test
+                  - name: HF_HOME
+                    value: /root/.cache/huggingface
+                  - name: VLLM_SOURCE_CODE_LOC
+                    value: /workspace/build/buildkite/vllm/performance-benchmark
+                  - name: HF_TOKEN
+                    valueFrom:
+                      secretKeyRef:
+                        name: hf-token-secret
+                        key: token
+                  - name: TEST_SELECTOR
+                    value: "llama70B"
+
+  
+  # - label: "A100 trt benchmark"
+  #   priority: 100
+  #   agents:
+  #     queue: A100
+  #   plugins:
+  #     - kubernetes:
+  #         podSpec:
+  #           <<: *common_pod_spec
+  #           containers:
+  #             - image: nvcr.io/nvidia/tritonserver:24.07-trtllm-python-py3
+  #               <<: *common_container_settings
+
+
+  # - label: "A100 tgi benchmark"
+  #   priority: 100
+  #   agents:
+  #     queue: A100
+  #   plugins:
+  #     - kubernetes:
+  #         podSpec:
+  #           <<: *common_pod_spec
+  #           containers:
+  #             - image: ghcr.io/huggingface/text-generation-inference:2.2.0
+  #               <<: *common_container_settings
```

Comment:
cleanup unless we need to keep them
```suggestion
```

Reply:
TGI benchmark: I prefer keeping it so that we can restore them after they support `--ignore-eos` flag.

TRT benchmark: I also prefer keeping it --- currently I am separating the trt-llm test to llama8B and llama 70B test and comment out this test purely because TRT needs to compile the model and it exceeds the 1 hour 30 minutes CI time limit if I directly run the commented-out test. I am expecting that new TRT-LLM docker will have all model for the test suite pre-compiled (hopefully) so that I don't need to worry about this test exceeding the test limit (and then I can uncomment this test).

Reply:
Just added comments in the file to explain why keeping those comments